src/app.js

```diff
@@ -0,0 +1,26 @@
+import renderImage from './render-image.js';
+import images from '../assets/images.js';
+import htmlToDOM from './utils/html-to-DOM.js';
+import renderHeader from '../src/render-header.js';
+
+const horns = document.querySelector('.horns');
+const body = document.getElementById('body');
+
+// STILL WORKING ON STRETCH GOAL
+// const radioInput = document.getElementsByName('horns');
+// const filterButton = document.getElementById('filter-button');
+
+// filterButton.addEventListener('click', () => {
+//     console.log(radioInput.value);
+// });
+
+const headerHtml = renderHeader();
+const headerDom = htmlToDOM(headerHtml);
+body.prepend(headerDom);
+
+images.forEach(function(item) {
```

Comment:
because this is an anonymous function, you can just put "images.forEach(image => {..." etc etc

Reply:
Gotcha. Thanks!